bundles/automation/org.eclipse.smarthome.automation.module.script.defaultscope/src/main/java/org/eclipse/smarthome/automation/module/script/defaultscope/internal/DefaultScriptScopeProvider.java

```diff
@@ -60,6 +67,7 @@
  * @author Simon Merschjohann - refactored to be an ScriptExtensionProvider
  *
  */
+@Component(immediate = true)
```

Comment:
We don't need immediate here, do we?
I see it has been immediate before, but as you changed from XML to annotations I think it is a good time to question it.

Reply:
I agree that it might not be needed, but as this is a last-minute change that I'd like to get into the openHAB 2.4 release, I prefer to keep it the same as before to reduce risk. Happy to remove it in a week!